modules/flowable-engine/src/test/java/org/flowable/standalone/validation/DefaultProcessValidatorTest.java

```diff
@@ -340,40 +336,41 @@ void testEventSubProcessWithBoundary() {
         assertThat(errors)
                 .extracting(ValidationError::getProblem, ValidationError::getDefaultDescription, ValidationError::getActivityId, ValidationError::isWarning)
                 .containsExactlyInAnyOrder(
-                        tuple(Problems.EVENT_SUBPROCESS_BOUNDARY_EVENT, "event sub process cannot have attached boundary events", "errorEndEventSubProcess", true)
+                        tuple(Problems.EVENT_SUBPROCESS_BOUNDARY_EVENT, "event sub process cannot have attached boundary events", "errorEndEventSubProcess",
+                                true)
                 );
     }
 
     protected void assertCommonProblemFieldForActivity(ValidationError error) {
         assertProcessElementError(error);
 
-        Assert.assertNotNull(error.getActivityId());
-        Assert.assertNotNull(error.getActivityName());
+        assertThat(error.getActivityId()).isNotNull();
+        assertThat(error.getActivityName()).isNotNull();
 
-        Assert.assertTrue(error.getActivityId().length() > 0);
-        Assert.assertTrue(error.getActivityName().length() > 0);
+        assertThat(error.getActivityId().length() > 0).isTrue();
+        assertThat(error.getActivityName().length() > 0).isTrue();
```

Comment:
Can be replaces with 

```
assertThat(error.getActivityId()).isNotEmpty();
assertThat(error.getActivityName()).isNotEmpty();
```

Reply:
Fixed; thanks; guess I was getting a little cross eyed by this time.